src/main/java/com/bytebreak/animagic/texture/AnimagicTextureRegion.java

```diff
@@ -5,65 +5,47 @@
 import com.badlogic.gdx.math.Vector2;
 
 public class AnimagicTextureRegion extends TextureRegion {
+    public AnimagicTextureData meta = new AnimagicTextureData();
```

Comment:
This is what my email earlier today was about. If we do this, things are easy to strap into this if we ever want to change what all is handled in the metadata.